src/components/Camera/PhotoPreview.js

```diff
@@ -44,7 +48,11 @@ const PhotoPreview = ( { photoUris, setPhotoUris }: Props ): Node => {
   };
 
   const emptyDescription = ( ) => (
-    <Text style={textStyles.topPhotoText}>
+    <Text style={[
+      textStyles.topPhotoText, {
+        bottom: 16 + insets.top
```

Comment:
Kind of the same thing here. You could just allow the style to set this value once with `bottom: textStyles.topPhotoText.bottom + insets.top`